v2/pkg/protocols/http/operators.go

```diff
@@ -121,7 +121,12 @@ func (request *Request) responseToDSLMap(resp *http.Response, host, matched, raw
 	data["response"] = rawResp
 	data["status_code"] = resp.StatusCode
 	data["body"] = body
-	data["content_length"] = resp.ContentLength
+	if resp.ContentLength > -1 {
```

Comment:
Can we pack this into a standard helper function that calculates the correct int64 length, for example, at `github.com/projectdiscovery/nuclei/v2/pkg/protocols/http/utils.go`?